pkg/config/config_local_test.go

```diff
@@ -316,4 +316,16 @@ var _ = Describe("Config Local", func() {
 		gomega.Expect(err).To(gomega.BeNil())
 		gomega.Expect(config2.Engine.MachineEnabled).To(gomega.Equal(true))
 	})
+
+	It("Rootless networking", func() {
+		// Given
+		config, err := NewConfig("")
+		gomega.Expect(err).To(gomega.BeNil())
+		gomega.Expect(config.RootlessNetworking()).To(gomega.Equal("slirp4netns"))
```

Comment:
To test the config is actually set, and not just returning from function, use
```suggestion
		gomega.Expect(config.Containers.RootlessNetworking).To(gomega.Equal("slirp4netns"))
```